main.py

```diff
@@ -1,7 +1,58 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+
+    genres = ["Western", "Action", "Dramma"]
+
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for actor in actors:
```

Comment:
Perform unpacking during iteration, e.g.
```suggestion
    for first_name, last_name in actors:
```